index.md

```diff
@@ -1,131 +1,198 @@
+defense and lavish projects while families go hungry and without shelter. We must hold governments accountable 
+to provide basic human services for all of their citizens. 
 
+Health, education and shelter should never be for-profit services and we should spend more on these services
+than we spend on defense and war.
 
-Cooperation > Competition
+
+Cooperation > competition
 -------------------------
 
-We have enough resources in this world to cover everyone's basic needs. But drastic inequality prevents this from happening as some people hoard resources and money. We need to reduce the inequality and free up resources to cover everyone’s basic needs. Especially now during a pandemic, we need to freely share medical information and cooperate on treatments, vaccines and medical devices in order to save lives.
+We have enough resources in this world to cover everyone's basic needs. But drastic inequality prevents this 
+from happening as some people hoard resources and money. We need to reduce the inequality and free up resources 
+to cover everyone’s basic needs. Especially now during a pandemic, we need to freely share medical information and 
+cooperate on treatments, vaccines and medical devices in order to save lives.
 
-Competing for resources can often be wasteful. How can we focus on cooperating in order to cover the basic needs of all people? The open source movement that produces much of the software that makes the Internet possible is created via global cooperation. This cooperation consumes far fewer resources than a competitive model would consume. How can we cooperate more and compete less?
+Competing for resources can often be wasteful. How can we focus on cooperating in order to cover the basic needs 
+of all people? The open source movement that produces much of the software that makes the Internet possible is created 
+via global cooperation. This cooperation consumes far fewer resources than a competitive model would consume. How 
+can we cooperate more and compete less?
 
 
 Focus on sharing and leadership
 -------------------------------
 
-Many people idolize the wrong role models. Many of the people whom society looks up to prey on the poor, disrespect other people and focus on questionable human values.
+Many people idolize the wrong role models. Many of the people whom society looks up to prey on the poor, disrespect 
+other people and focus on questionable human values.
 
-If these role models are hoarding resources and actively creating inequality in the world, why do we look up to them? Hoarding resources does not stimulate the economy and when you have more than you need to live well and you are not sharing these resources, you are violating basic human decency. Being disrespectful or preying on others are not human qualities we should look up to or reward.
+If these role models are hoarding resources and actively creating inequality in the world, why do we look up to them? 
```

Comment:
Same here, either "these role models are hoarding resources and actively creating inequality in the world" or not, what's the point on asking "If". Those models are clearly responsible on many behaviors leading to more inequality, there's no question about it imho.